definitions/checks/check_hotfix_installed.rb

```diff
@@ -12,8 +12,8 @@ class Checks::CheckHotfixInstalled < ForemanMaintain::Check
   end
 
   def run
-    if feature(:downstream) && feature(:downstream).subscribed_using_activationkey?
-      skip 'Your system is subscribed using custom activationkey'
+    if feature(:downstream).subscribed_using_activationkey?
+      skip "Hotfixes can't be detected"
```

Comment:
@kgaikwad, sorry for the confusion, I think the original part is important too:
`Your system is subscribed using custom activationkey. Hotfixes can't be detected.`.
This way the user knows the cause and can understand the result.

Reply:
No problem! my bad I was thinking the same :-) but then I thought you expected this as main message.I will do that